package.json

```diff
@@ -8,7 +8,7 @@
   "scripts": {
     "analyze": "webpack --mode=none",
     "prebuild:dev": "yarn clean && node generate_translations.js",
-    "build:dev": "webpack --env COMMIT_HASH=$(git rev-parse --short HEAD) --mode=development",
+    "build:dev": "webpack --env COMMIT_HASH=DEV --mode=development",
```

Comment:
Why this change?

Reply:
I added that by accident I had to do that to make it build on windows.

Reply:
You need to revert that then.